Dockerfile

```diff
@@ -27,7 +27,7 @@ RUN apt-get update && apt install -y libcap2-bin
 RUN setcap CAP_NET_BIND_SERVICE=+ep /usr/local/bin/envoy
 RUN setcap CAP_NET_BIND_SERVICE=+ep /usr/local/bin/$BIN_NAME
 
-FROM hashicorp/envoy-fips:1.28.1-fips1402 as envoy-fips-binary
+FROM hashicorp/envoy-fips:1.28.2-fips1402 as envoy-fips-binary
```

Comment:
I think the tests are failing due to the `hashicorp/envoy-fips:1.28.2-fips1402` container image that doesn't yet exist. Not sure on how to publish a new one, some guidance would be helpful 👀

Reply:
That's correct. There's an effort right now to reconfigure ownership of that publishing going forward; I'll drop a question in Slack to see if we can get a hand. For now, @dduzgun-security I'd omit this and put in a TODO that it needs to be bumped before the next release (that's what we did last time)

Reply:
Thanks for initiating the Slack conversation, I have reverted the change: https://github.com/hashicorp/consul-dataplane/pull/474/commits/90790c1267c76b964c669dabcf05d20db449bd6a and added ticket [NET-9002] to revisit it before the next release.